plugins/MumblePlugin.h

```diff
@@ -129,7 +129,7 @@
  * MUMBLE_QUOTE
  * Quotes the given value WITHOUT macro expansion
  */
-#	define MUMBLE_QUOTE(arg) #    arg
+#	define MUMBLE_QUOTE(arg) # arg
```

Comment:
The formatting CI job will probably complain about this change (likely introduced by using a different clang-format version that v10)

Reply:
I changed it to the previous version.